docker/bootstrap/files/pre_stop.sh

```diff
@@ -17,18 +17,18 @@
 
 set -e
 
-echo "running PreStop Hook" > /proc/1/fd/1
-echo "**disablegossip**" > /proc/1/fd/1
+echo "running PreStop Hook"
+echo "**disablegossip**"
 nodetool disablegossip
 sleep 1
-echo "**disablebinary**" > /proc/1/fd/1
```

Comment:
I remove this because we can't use it in ReadOnluy Container.
But this is no more logs in the output of the container.
@cscetbon If you think at a solution to allow to log that in the output that would be great :)